Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-emailthreatsubmissionpolicy-list.md

---
title: "List emailThreatSubmissionPolicies"
description: "Get a list of the emailThreatSubmissionPolicy objects and their properties."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# List emailThreatSubmissionPolicies
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Get a list of the [emailThreatSubmissionPolicy](../resources/security-emailthreatsubmissionpolicy.md) objects and their properties.

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmission.Read, ThreatSubmission.ReadWrite, ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All, ThreatSubmissionPolicies.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported|
|Application|ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All, ThreatSubmissionPolicy.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET security/threatSubmission/emailThreatSubmissionPolicies
```

## Optional query parameters

> Not suppported by now since only 1 email threat submission policy is supported by now and it is designed for future usage.

Reviewer: ```suggestion
Not supported as of today since only one email threat submission policy is supported as of today.  It is designed for future usage.
```

> This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).

Reviewer 2: In general, it provides a good experience when we list or mention which query parameters are supported. If the filter parameter is supported, then please indicate which properties can be filtered.

Reviewer 2: Please apply to all List and Get topics in this PR.

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body

> Do not supply a request body for this method.

Reviewer: ```suggestion
A request body is not required for this method. 
```
